buildSrc/src/main/resources/unixStartScript.txt

```diff
@@ -165,7 +165,12 @@ case "\$( uname )" in                #(
   NONSTOP* )        nonstop=true ;;
 esac
 
-CLASSPATH=$classpath
+if [ -z "\$EXTRA_CLASSPATH" ]; then
+    CLASSPATH=$classpath
+else
+    CLASSPATH="\$EXTRA_CLASSPATH":$classpath
+fi
```

Comment:
A bit nit-picky, but can you add this comment and try this combined version with your test setup? (I've verified to it seems to work for me.) `$classpath` is potentially the majority of the script length and if we can do without doubling it that would be good.

```shell
# Deephaven edit.
# Prepend EXTRA_CLASSPATH and colon if EXTRA_CLASSPATH is non-empty.
CLASSPATH=\${EXTRA_CLASSPATH:+\$EXTRA_CLASSPATH:}$classpath
```